mne/tests/test_line_endings.py

```diff
@@ -0,0 +1,66 @@
+# Adapted from vispy
+#
+# License: BSD (3-clause)
+
+import os
+from nose.tools import assert_raises
+from nose.plugins.skip import SkipTest
+from os import path as op
+import sys
+
+import mne
+from mne.utils import run_tests_if_main, _TempDir
+
+
+skip_files = (
+    # known crlf
+    'FreeSurferColorLUT.txt',
+    'test_edf_stim_channel.txt',
+    'FieldTrip.py',
```

Comment:
It's not standard practice to have CRLF in `.py` files on *nix systems, so I don't mind having to call out the offenders individually. I only had to skip this one to get it to work.